experimental/devices/as7262/as7262.go

```diff
@@ -0,0 +1,532 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// +build go1.7
+
+package as7262
+
+import (
+	"context"
```

Comment:
Eventually I'd prefer this to be rewritten without using context (even if your use is valid, mostly for "lightweightness" and support for old versions). This is fine for now.

Reply:
As above, it's not using any child ctx so it could be replaced with a simpler cancellation scheme, so that it still builds with go1.5.

Reply:
see #301